code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -456,9 +456,12 @@ var/global/list/damage_icon_parts = list()
 
 	//base icons
 	var/icon/hair_standing	= new /icon('icons/mob/human_face.dmi',"bald_s")
+	//var/icon/debrained_s = new /icon("icon"='icons/mob/human_face.dmi', "icon_state" = "debrained_s")

```

Comment:
It works, but it doesn't properly update, so...Fethas left it in for someone else to get working. I don't have a strong opinion on *this* particular section--but I do agree that other commented code should be removed.